### ISSUE TITLE
Multiple Administrator

### ISSUE THREAD

**Issue by benyaminl** [serghey-rodin/vesta#2160]
Hello. I'm kind of wondering, is there any roadmap or grand design that would make vesta available to multi admin environment? As some people use it to share admin user password, but it will be hard if one of the person resign from the company, is there any plan to make Vesta able to work with multi admin, and if it's not, will vesta open to this kind of suggestion and pull request? 

Thank you

**Comment by pashtet422** [comment]
Hello.unfortunately there is no way to make multiple administrators in vesta. and as u can see maintainers wont release anything for a log period of time (until this month). So if u are looking for an easiest way to solve your issue u should take a look to vesta fork, called [HestiaCP] (https://www.hestiacp.com/). this feature works out of the box and u can choose a role for every single user. also hestia has some killing features (like multiple php, correctly working jail) and so on. cheers

**Comment by benyaminl** [comment]
> Hello.unfortunately there is no way to make multiple administrators in vesta. and as u can see maintainers wont release anything for a log period of time (until this month). So if u are looking for an easiest way to solve your issue u should take a look to vesta fork, called [HestiaCP] (https://www.hestiacp.com/). this feature works out of the box and u can choose a role for every single user. also hestia has some killing features (like multiple php, correctly working jail) and so on. cheers

Well I wanted to but will lot of server farm, and diversing version between them I just stick with Vesta and opt to contribute on it. Just I need someone from vesta to accept if they're a pull request regarding that. 

Anyway I think they on hiatus because they redesign the whole system ground up.

**Comment by pashtet422** [comment]
backups are compatible between them, if u know what i mean. anyway good luck to you. i thought that vesta was abandoned

**Comment by anton-reutov** [comment]
**benyaminl** it's great idea. Can you create issue with this idea ?

**Comment by anton-reutov** [comment]
Are you need this changes on your server ?
https://forum.vestacp.com/viewtopic.php?t=10752#p40140

**Comment by benyaminl** [comment]
> Are you need this changes on your server ? https://forum.vestacp.com/viewtopic.php?t=10752#p40140

I did already put on my server on dutch tape for now. But I think we need to implement multi admin, so Vesta can be fully equipped as fully fledged Web Panel for enterprise. At least bear up webuzo or cpanel in near future. 



> backups are compatible between them, if u know what i mean. anyway good luck to you. i thought that vesta was abandoned

I did try spin up with the image from the vesta to hestia, sadly it crash, and I don't intend to talk about hestiacp here, even it's file manager panel is broken to the point can't be used. So I opt out. Vesta still alive, and I will kept using it.

**Comment by benyaminl** [comment]
> **benyaminl** it's great idea. Can you create new issue with this idea ?

I think this is already new Issue, do you mean I close this issue and open new one here?

**Comment by jaapmarcus** [comment]
I hope this is not the method you are going to implement... 

Manual edit that get overwritten at each update...

Consider at least a implementation where you can set the "role" in user settings 

https://github.com/hestiacp/hestiacp/blob/303272721aa51f9e131767417a73fae13e16c02c/web/login/index.php#L114-L118

**Comment by anton-reutov** [comment]
> I think this is already new Issue, do you mean I close this issue and open new one here?
yes, please open new

**Comment by victor-50** [comment]
I made the mistake to add a user, and as I saw no choice for a role I assumed it was an administrator. Am not sure anymore but I think I deleted the admin, but maybe that wasn't possible. Took me a long time and correspondence with the supplier of my VPS to find out that I was not an administrator and had a limited menu. Only after reinstalling the VPS I could go on. Then it turns out you can't rename "admin". Not so good as it makes hacking attempts easier. On the side, I don't like the automatic admin_ prefix when you set up a database.

**Comment by jaapmarcus** [comment]
Create a new user in VestaCP you can rename / remove the admin user it *** the whole system.
